packages/components-react/projects/nextjs/components/Select.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
+
+import { routes } from '../routes';
+import { useRouter, usePathname } from 'next/navigation';
+
+export const Select = (): JSX.Element => {
+  const router = useRouter();
+  const pathname = usePathname();
+  const selected: string = pathname === null ? '' : pathname;
```

Comment:
This should achieve the same but shorter and future prove.
```suggestion
  const selected: string = pathname ?? '';
```